tasks/rh-sign-image/tests/test-rh-sign-image-single-component-plr.yaml

```diff
@@ -0,0 +1,190 @@
+        - name: data
+          workspace: tests-workspace
+      taskSpec:
+        steps:
+          - name: check-result
+            image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+            script: |
+              #!/usr/bin/env bash
+              set -eux
+
+              internalPipelineruns="$(kubectl get pr \
+                -l "internal-services.appstudio.openshift.io/pipelinerun-uid=$(context.pipelineRun.uid)" \
+                -o json --sort-by=.metadata.creationTimestamp | jq -c)"
+              plrsLength=$(jq ".items | length" <<< "${internalPipelineruns}" )
+
+              expectedReferences=()
+              expectedReferences+=("registry.redhat.io/myproduct/myrepo:some-prefix")
+              expectedReferences+=("registry.access.redhat.com/myproduct/myrepo:some-prefix")
+
+              expectedDigests=()
+              expectedDigests+=("sha256:0000")
+              expectedDigests+=("sha256:0000")
+
+              foundReferences=()
+              foundDigests=()
+              for((iplr=0; iplr<plrsLength; iplr++)); do
+                params=$(jq -r ".items[$iplr].spec.params" <<< "${internalPipelineruns}")
+                refValue=$(jq -r '.[1].value' <<< "${params}")
+                manifestDigest=$(jq -r '.[2].value' <<< "${params}")
+                foundReferences+=("${refValue}")
+                foundDigests+=("${manifestDigest}")
+
+                if [ "$(jq -r '.[4].value' <<< "${params}")" != "signing-config-map" ]; then
+                  echo "config_map_name does not match"
+                  exit 1
+                fi
+
+                if [ "$(jq -r '.[3].value' <<< "${params}")" != "testuser-single" ]; then
+                  echo "requester does not match"
+                  exit 1
+                fi
+
+                if [ "$(jq -r '.[0].value' <<< "${params}")" != \
+                   "quay.io/redhat-isv/operator-pipelines-images:released" ]; then
+                  echo "pipeline_image does not match"
+                  exit 1
+                fi
+              done
+
+              differenceReferences=$(echo "${expectedReferences[@]}" "${foundReferences[@]}" | tr ' ' '\n' \
+                | sort | uniq -u)
+              differenceDigests=$(echo "${expectedDigests[@]}" "${foundDigests[@]}" | tr ' ' '\n' | sort | uniq -u)
+
+              if [ -n "${differenceReferences}" ] ; then
+                echo "error: references expected compared to found do not match"
+                echo ""
+                echo "${differenceReferences}"
+                exit 1
+              fi
+              if [ -n "${differenceDigests}" ] ; then
+                echo "error: digests expected compared to found do not match"
+                echo ""
+                echo "${differenceDigests}"
+                exit 1
+              fi
+      runAfter:
+        - run-task
+  finally:
+    - name: cleanup
+      taskSpec:
+        steps:
+          - name: delete-crs
```

Comment:
similar comment about this task

Reply:
corrected task name and implemented plr deletion.